src/components/itemContextMenu.js

```diff
@@ -466,6 +477,15 @@ function executeCommand(item, id, options) {
             case 'edit':
                 editItem(apiClient, item).then(getResolveFunction(resolve, id, true), getResolveFunction(resolve, id));
                 break;
+            case 'editplaylist':
+                import('./playlisteditor/playlisteditor').then(({ default: PlaylistEditor }) => {
+                    const playlistEditor = new PlaylistEditor();
+                    playlistEditor.show({
+                        id: itemId,
+                        serverId
+                    }).then(getResolveFunction(resolve, id, true), getResolveFunction(resolve, id));
+                });
```

Comment:
Isn't this more readable without chaining thens?

Reply:
I'm not sure I follow what you are suggesting 😅